ui/v2.5/src/docs/en/Manual/Scraping.md

```diff
@@ -30,18 +30,51 @@ Stash provides the following built-in scrapers:
 | Freeones | `search` Performer scraper for freeones.xxx. |
 | Auto Tag | Scene `fragment` scraper that matches existing performers, studio and tags using the filename. |
```

Comment:
Does the new system impact these in any way?

Reply:
Nope. These scrapers are always available and not managed by the new system.